python/ray/autoscaler/_private/util.py

```diff
@@ -722,6 +722,20 @@ def get_demand_report(lm_summary: LoadMetricsSummary):
     return demand_report
 
 
+def get_per_node_breakdown_as_dict(
+    lm_summary: LoadMetricsSummary,
+) -> dict:
+    per_node_breakdown = {}
+
+    for node_id, usage in lm_summary.usage_by_node.items():
+        usage_string = ""
+        for line in parse_usage(usage, verbose=True):
+            usage_string += f"{line}\n"
+        per_node_breakdown[node_id] = usage_string.strip()
+
+    return per_node_breakdown
```

Comment:
Can you add a unit test for this function?

Reply:
Sure